plugins/extensions/plugins/jsonSerialization.h

```diff
@@ -444,7 +444,8 @@ void init(brayns::AnimationParameters* a, ObjectHandler* h)
 namespace brayns
 {
 template <class T, class F>
-inline bool from_json(T& obj, const std::string& json, F postUpdateFunc = [] {})
+inline bool from_json_(T& obj, const std::string& json,
```

Comment:
Is there a reason for the added underscore? :thinking:

Reply:
It caused ambiguities in the test when reusing the from_json function from below. This one is not meant to be reused anywhere else. I probably should move out it of here for clarity.